tests/unit/backend/helpers/jest-db.ts

```diff
@@ -222,13 +230,45 @@ const getFullFormById = async (
 ): Promise<IPopulatedForm | null> =>
   await getEmailFormModel(mongoose).getFullFormById(formId)
 
+const insertVerifiedSms = async ({
+  formId,
+  formAdmin,
+  msgSrvcSid = 'MOCK MESSAGE SERVICE ID',
+  logType = LogType.success,
+  isOnboarded = false,
+}: {
+  formId: ObjectID
+  formAdmin: { email: string; userId: ObjectID }
+  msgSrvcSid?: string
+  logType?: LogType
+  isOnboarded?: boolean
+}): Promise<ISmsCountSchema> => {
+  const SmsModel = getSmsCountModel(mongoose)
+  const sms = await SmsModel.create({
+    smsType: SmsType.Verification,
+    logType,
+    msgSrvcSid,
+    form: formId,
+    formAdmin,
+    isOnboarded,
```

Comment:
Should is `isOnboarded` be skipped in the create parameter? Seems to do nothing. Was this put in erroneously?

Reply:
made a mistake here, will change in the create portion! thanks for the catch!